src/components/yeti/ElevationProfile.vue

```diff
@@ -0,0 +1,386 @@
+<template>
```

Comment:
there are diffs between c2c's elevation profile and this one, but maybe we could build some factorized code? I assume you thought of it bu voted against?